drivers/led/issi/is31fl3729-mono.h

```diff
@@ -0,0 +1,293 @@
+/* Copyright 2023 HorrorTroll <https://github.com/HorrorTroll>
+ * Copyright 2023 Harrison Chan (Xelus)
+ * Copyright 2023 Dimitris Mantzouranis <d3xter93@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "progmem.h"
+#include "util.h"
+
+#define IS31FL3729_REG_SCALING 0x90
+#define IS31FL3729_REG_CONFIGURATION 0xA0
+#define IS31FL3729_REG_GLOBAL_CURRENT 0xA1
+#define IS31FL3729_REG_PULLDOWNUP 0xB0
+#define IS31FL3729_REG_SPREAD_SPECTRUM 0xB1
+#define IS31FL3729_REG_PWM_FREQUENCY 0xB2
+#define IS31FL3729_REG_RESET 0xCF
+
+#define IS31FL3729_I2C_ADDRESS_GND 0x34
+#define IS31FL3729_I2C_ADDRESS_SCL 0x35
+#define IS31FL3729_I2C_ADDRESS_SDA 0x36
+#define IS31FL3729_I2C_ADDRESS_VCC 0x37
+
+#if defined(LED_MATRIX_IS31FL3729)
+#    define IS31FL3729_LED_COUNT LED_MATRIX_LED_COUNT
+#endif
+
+#if defined(IS31FL3729_I2C_ADDRESS_4)
+#    define IS31FL3729_DRIVER_COUNT 4
+#elif defined(IS31FL3729_I2C_ADDRESS_3)
+#    define IS31FL3729_DRIVER_COUNT 3
+#elif defined(IS31FL3729_I2C_ADDRESS_2)
+#    define IS31FL3729_DRIVER_COUNT 2
+#elif defined(IS31FL3729_I2C_ADDRESS_1)
+#    define IS31FL3729_DRIVER_COUNT 1
+#endif
+
+typedef struct is31fl3729_led_t {
+    uint8_t driver : 2;
+    uint8_t v;
+} PACKED is31fl3729_led_t;
+
+extern const is31fl3729_led_t PROGMEM g_is31fl3729_leds[IS31FL3729_LED_COUNT];
+
+void is31fl3729_init_drivers(void);
+void is31fl3729_init(uint8_t addr);
+void is31fl3729_write_register(uint8_t addr, uint8_t reg, uint8_t data);
+bool is31fl3729_write_pwm_buffer(uint8_t addr, uint8_t *pwm_buffer);
```

Comment:
```suggestion
```

Reply:
Thanks, I've just push a new commit to resolved this.